common/source/docs/common-learning-a-mission.rst

```diff
@@ -0,0 +1,24 @@
+.. _common-learning-a-mission:
+
+==================
+Learning a Mission
```

Comment:
I wonder if it's "Learning" or perhaps more accurately, "Recording"?

Reply:
done